<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingTab</class>
 <widget class="QWidget" name="BookingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>740</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Booking Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Customer:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="customerCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Screening:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="screeningCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Booking Date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="bookingDateInput"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Total Amount:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="totalAmountInput">
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="bookingTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Booking ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Customer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Movie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Screening Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Booking Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Amount</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addBookingBtn">
       <property name="text">
        <string>Add Booking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateBookingBtn">
       <property name="text">
        <string>Update Booking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBookingBtn">
       <property name="text">
        <string>Delete Booking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBookingBtn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
